uber/models/__init__.py

```diff
@@ -1487,10 +1491,6 @@ def insert_test_admin_account(self):
 
             self.add(all_access_group)
             self.add(test_developer_account)
-            self.add(AdminAccessGroup(
```

Comment:
Removing this will create the admin but give them no permissions to view anything whatsoever. You... probably do not want that.

FWIW, this table DOES exist. It's here: https://github.com/magfest/ubersystem/blob/master/uber/models/admin.py#L24 It's a many-to-many table defined a little differently than normal, so that's why you can't find it by the python class name. I was under the impression that this block of code worked, but if it doesn't, we'll need to figure something else out.